cmd/kubeadm/app/util/default.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"k8s.io/klog"
+)
+
+func WarnDefaultComponentConfig(paramName string, componentName string, defaultValue interface{}) {
```

Comment:
Again, this should be in `cmd/kubeadm/app/componentconfigs/defaults.go` and package private.

Reply:
Thanks, Fixed.